lib/Server.js

```diff
@@ -378,57 +378,30 @@ class Server {
     return /^[a-zA-Z][a-zA-Z\d+\-.]*:/.test(URL);
   }
 
-  /**
-   * @param {string} gateway
-   * @returns {string | undefined}
-   */
-  static findIp(gateway) {
```

Comment:
We need to keep this function, because developer still can use it, otherwise it is a breaking change

Reply:
ok, I'll change my changes.

Reply:
done